.github/workflows/publish-dev.yml

```diff
@@ -95,44 +76,31 @@ jobs:
-          docker push -a ghcr.io/${{ github.repository }}/dev_vro-db-init
-          docker push -a ghcr.io/${{ github.repository }}/dev_vro-service-pdfgenerator
-          docker push -a ghcr.io/${{ github.repository }}/dev_vro-service-assessclaimdc7101
-          docker push -a ghcr.io/${{ github.repository }}/dev_vro-service-assessclaimdc6602
-          docker push -a ghcr.io/${{ github.repository }}/dev_vro-service-data-access
+          source scripts/image_vars.src
 
+          # Push dev images for non-prod testing
+          for PREFIX in ${VAR_PREFIXES_ARR[@]}; do
+            IMG_NAME=dev_$(getVarValue ${PREFIX} _IMG)
+            echo "Pushing image '$IMG_NAME'"
+            docker push -a "ghcr.io/${{ github.repository }}/${IMG_NAME}"
+          done
       - name: Cleanup images
         run: |
-          # Reminder to also add new images to build.yml and .github/secrel/config.yml
-
-          # Push the following images to work around https://www.docker.com/increase-rate-limits/
-          docker rmi "ghcr.io/${{ github.repository }}/vro-rabbitmq:3"
-
-          # Pushes dev images for non-prod testing
-          docker rmi ghcr.io/${{ github.repository }}/dev_vro-postgres:${COMMIT_SHA:0:7} \
-                     ghcr.io/${{ github.repository }}/dev_vro-app:${COMMIT_SHA:0:7} \
-                     ghcr.io/${{ github.repository }}/dev_vro-db-init:${COMMIT_SHA:0:7} \
-                     ghcr.io/${{ github.repository }}/dev_vro-service-pdfgenerator:${COMMIT_SHA:0:7} \
-                     ghcr.io/${{ github.repository }}/dev_vro-service-assessclaimdc7101:${COMMIT_SHA:0:7} \
-                     ghcr.io/${{ github.repository }}/dev_vro-service-assessclaimdc6602:${COMMIT_SHA:0:7} \
-                     ghcr.io/${{ github.repository }}/dev_vro-service-data-access:${COMMIT_SHA:0:7} \
-                     ghcr.io/${{ github.repository }}/dev_vro-postgres:latest \
-                     ghcr.io/${{ github.repository }}/dev_vro-app:latest \
-                     ghcr.io/${{ github.repository }}/dev_vro-db-init:latest \
-                     ghcr.io/${{ github.repository }}/dev_vro-service-pdfgenerator:latest \
-                     ghcr.io/${{ github.repository }}/dev_vro-service-assessclaimdc7101:latest \
-                     ghcr.io/${{ github.repository }}/dev_vro-service-assessclaimdc6602:latest \
-                     ghcr.io/${{ github.repository }}/dev_vro-service-data-access:latest
-      
+          source scripts/image_vars.src
+
+          # Clean up dev images locally
+          for PREFIX in ${VAR_PREFIXES_ARR[@]}; do
+            IMG_NAME=dev_$(getVarValue ${PREFIX} _IMG)
+            echo "Clean up image with tags '$IMG_NAME:${COMMIT_SHA:0:7}' and '${IMG_NAME}:latest'"
+            docker rmi "ghcr.io/${{ github.repository }}/${IMG_NAME}:${COMMIT_SHA:0:7}" \
+                       "ghcr.io/${{ github.repository }}/${IMG_NAME}:latest"
+          done
```

Comment:
This step doesn't seem necessary but @advcoder thinks it may help mitigate other problems.